tests/configs.py

```diff
@@ -1,13 +1,12 @@
-from __future__ import absolute_import
-from __future__ import print_function
+
 import os
 from _universum.configuration_support import Variations
 
 env_name = "virtual_universe"
 
 
 def run_virtual(cmd):
-    return ["env", "-i", "PATH=" + os.getenv("PATH"), "bash", "-c", "source {}/bin/activate; {}".format(env_name, cmd)]
+    return ["env", "-i", "PATH=" + os.getenv("PATH"), "bash", "-c", f"source {env_name}/bin/activate; {cmd}"]
```

Comment:
Please remove fstrings from this file until final switching to Python3.7; instead please add 'python3.7 -m' into Variations